### ISSUE TITLE
Save actions for all projects

### ISSUE THREAD

**Issue by JVidiri** [dubreuia/intellij-plugin-save-actions#305]
**Describe the solution you'd like**
I want to be able to add save my save actions configurations for all projects in my workspace, the option today is just for current project.

**For what product**
Intellij

**Comment by dubreuia** [comment]
This was asked before, I won't have time to implement this, but you can submit a PR anytime.

You can check this PR: https://github.com/dubreuia/intellij-plugin-save-actions/pull/251, it was implemented by another user, but I wasn't happy with how it works.

**Comment by IsaacHub** [comment]
Yes, the global setting is what majority of the people needed, every time we open up a new project, we need to go to settings and enable "save actions" for every project, which is time-consuming and frustrating. For those who don't want it to be global, then what's the point of using this plugin?

**Comment by IsaacHub** [comment]
I downloaded this https://github.com/le-yams/intellij-plugin-save-actions and built the JAR file and placed in the plugins folder and now it has global config and project config. Happy Ending!

**Comment by dubreuia** [comment]
Helllo @IsaacHub, thank you for your interest in the plugin.

> Yes, the global setting is what majority of the people needed, every time we open up a new project, we need to go to settings and enable "save actions" for every project, which is time-consuming and frustrating. For those who don't want it to be global, then what's the point of using this plugin?

I don't decide which features are present in this plugin, per project settings where voted on multiple times and very early on (in 2017 with #68) by the users. I understand this is a feature you need, but we both know that claiming "majority of the people needed" is wrong because you haven't properly verified this. 

Happy to hear the @le-yams build works for you. I'll keep this issue open for further discussion with everybody.

**Comment by dubreuia** [comment]
I just found that there are are "For new project" settings that can be configured, so that every new project you start will have the same settings. It can be found in the "File > New Project Settings > Settings for new projects".

I think this is a good compromise, and I'm pretty sure it is easy to implement. Vote on this comment if you find that useful.

**Comment by dubreuia** [comment]
You have voted this proposition as sufficient, I will document it and close this isse

**Comment by AndroidDeveloperLB** [comment]
Where is it documented? It's on the "todo" list?

**Comment by dubreuia** [comment]
For now, it isn't documented anywhere, because I haven't done it yet, this issue is still open.

I've poked around the "For new project" settings recently, and I'm unsure we can do it this way (it seems new project settings apply to certain stuff, but not plugin settings). I'm reverting to the original title.

**Comment by AndroidDeveloperLB** [comment]
@dubreuia  Can you please at least consider to offer "export" and "import" as a workaround? Or this is available somewhere?

**Comment by dubreuia** [comment]
I won't have time to implement anything regarding this, but PRs are welcomed! You can also check the [files locations](https://github.com/dubreuia/intellij-plugin-save-actions#files-location), you can copy / commit the settings files which is in XML.

**Comment by AndroidDeveloperLB** [comment]
@dubreuia Sorry but I lack the knowledge of developing plugins for Android Studio. :(

**Comment by CodingSpiderFox** [comment]
This is the thing I am missing the most about this otherwise great plugin
